com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -393,6 +397,17 @@ half3 DirectBDRF(BRDFData brdfData, half3 normalWS, half3 lightDirectionWS, half
 //                      Global Illumination                                  //
 ///////////////////////////////////////////////////////////////////////////////
 
+// Samples the Ambient Occlusion texture
+half SampleScreenSpaceOcclusionTexture(half3 positionCS)
```

Comment:
instead of positionCS should we call this some normalized uv (screen space?)

Reply:
Yes, I've renamed the variable "normalizedScreenSpaceUV"